pkg/buses/message_dispatcher.go

```diff
@@ -66,14 +65,21 @@ func NewMessageDispatcher(logger *zap.SugaredLogger) *MessageDispatcher {
 // The destination and replyTo are DNS names. For names with a single label,
 // the default namespace is used to expand it into a fully qualified name
 // within the cluster.
-func (d *MessageDispatcher) DispatchMessage(message *Message, destination string, defaults DispatchDefaults) error {
-	destinationURL := d.resolveURL(destination, defaults.Namespace)
-	reply, err := d.executeRequest(destinationURL, message)
-	if err != nil {
-		return fmt.Errorf("Unable to complete request %v", err)
+func (d *MessageDispatcher) DispatchMessage(message *Message, destination, replyTo string, defaults DispatchDefaults) error {
```

Comment:
Talked with Adam offline about the signature change that moved `defaults.ReplyTo` into the positional parameter `replyTo`. He argued that `replyTo` is not actually a default since it's specific to each Subscription and is never overridden.